README.md

```diff
@@ -15,9 +15,10 @@ make images
 Then from your operator's repository run:
 
 ```
-docker run -v $(pwd)/manifests:/go/src/github.com/openshift/crd-schema-gen/manifests:Z \
-  -v $(pwd)/vendor/github.com/openshift/api:/go/src/github.com/openshift/crd-schema-gen/openshift/api:Z \
-  -e APIS_DIR="openshift/api" openshift/origin-crd-schema-gen
+docker run -v $(pwd)/manifests:/go/src/crd-schema-gen/manifests:Z \
+  -v $(pwd)/vendor:/go/src/crd-schema-gen/vendor:Z \
+  openshift/origin-crd-schema-gen \
+  --apis-dir vendor/github.com/openshift/api
```

Comment:
why so complicated? why not:
```bash
PKG=${PWD#${GOPATH}/}
docker run  -v ${PWD}:/go/${PKG} -w /go/${PKG} openshift/origin-crd-schema-gen --apis-dir ....
```
Mounting into a fake dir is strange.

Reply:
That is much simpler, didn't know I could set the workdir with a flag. Updated